.github/workflows/deploy-protocol-to-miannet.yaml

```diff
@@ -78,6 +78,11 @@ jobs:
           default_author: user_info
           message: '[skip ci] Deployed contracts to ${{ vars.HARDHAT_NETWORK }} Mainnet'
           fetch: --no-tags --recurse-submodules=no # Prevent failure due to submodules
+
+      - name: Verify deployment result
```

Comment:
Add `if: ${{ always() }}` condition to `publish-source-code` job. It will trigger publicashion even if deploy failed.
More info: https://docs.github.com/en/actions/using-jobs/using-jobs-in-a-workflow#example-not-requiring-successful-dependent-jobs